judgels-frontends/raphael/src/modules/api/uriel/contestProblem.ts

```diff
@@ -16,15 +16,15 @@ export interface ContestProblem {
   alias: string;
   problemJid: string;
   status: ContestProblemStatus;
-  submissionsLimit: number;
-  points?: number;
+  submissionsLimit: number | null;
```

Comment:
So actually, every `Optional<X>` in our backend response gets translated into `X | null` in TS, but because it is cumbersome, we always use `X?` instead.

Can we do the same for these two fields?

Reply:
(and skip the fields in tests if they are not present)

Reply:
Hm, so what should we  do when checking for the value? can I treat null as undefined?

Reply:
Hmm in this case we can just use `!!submissionsLimit` and `!!points` since 0 value is invalid anyway